### PR TITLE
Lightswitch fixes, fixes Quick join lights being off

### PR DESCRIPTION
**Pull request by Bod9001**
### Purpose
Removes Quantum nature of lights
#1989
### Open Questions and Pre-Merge TODOs

- [x]  This fix is tested on the same branch it is PR'ed to.
- [x]  I correctly commented my code
- [x]  My code is indented with tabs and not spaces
- [x]  This PR does not include any unnecessary .meta, .prefab or .unity (scene) changes
- [x]  This PR does not bring up any new compile errors
- [x]  This PR has been tested in editor
- [x]  This PR has been tested in multiplayer (with 2 clients and late joiners, if applicable)

### Notes:
_Please enter any other relevant information here_

### CODE REVIEW

File: UnityProject/Assets/Scripts/Electricity/PoweredDevices/APC.cs [hunk 1]
```diff
@@ -53,9 +53,18 @@ private void SetVoltage(float newVoltage)
 		UpdateDisplay();
 	}
 
+	 private void OnStartServer()
+	{
+		SetVoltage(this.voltageSync);
+
+		//if extending another component
+		base.OnStartServer();
+	}
```
Comment (Aranclanos) [inline UnityProject/Assets/Scripts/Electricity/PoweredDevices/APC.cs:62]: remember that OnStartClient() is run on the server

Comment (Bod9001) [reply UnityProject/Assets/Scripts/Electricity/PoweredDevices/APC.cs:62]: It should be fine , Since it's Setting to the same value?

**Comment by DooblyNoobly** [comment]
@Aranclanos is this resolved? Bod has marked it as that.

**Comment by Aranclanos** [comment]
not really but whatevs

**Comment by Bod9001** [comment]
well, It's said as good practice on syncvars 
https://github.com/unitystation/unitystation/wiki/SyncVar-Best-Practices-for-Easy-Networking

Proper SyncVar Usage No. 3

**Pull request merged**